### ISSUE TITLE
"Skip to content" link

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#2095]
Content that appears in the same way on most pages, such as the main navigation, should have a bypass link (skiplink). This is generally the first link you see when you open a new page and press ‘tab’ on keyboard.
For large content blocks such as the tags, a skiplink could also be a welcome addition.

See https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks
